### PR TITLE
Changes fix in preparation to 11.0

### PR DESCRIPTION
**Pull request by gasche**
See #176.

### CODE REVIEW

File: Changes [hunk 1]
```diff
@@ -14,31 +14,12 @@ of the usual "- ".
...
-OCamlbuild 0.10.0 introduces a minor change to the way `.cmxs` files
-are produced when no `.mldylib` file is absent: it will now use the
-exact same semantics as `.cmxa` and `.mllib` file -- in particular, it
+OCamlbuild 0.11.0 introduces a change to the way `.cmxs` files are
+produced when no `.mldylib` file is absent: it will now use the exact
+same semantics as `.cmxa` and `.mllib` file -- in particular, it
 should not be necessary anymore to have identical
 `foo.{mllib,mldylib}` files, only `foo.mllib` should suffice. See the
-detailed Changelog for details.
+detailed Changelog below for details.
```
Comment (whitequark) [inline Changes]: Nit: the file is called "Changes" (was it not?), so this should probably be "detailed change log".

Comment (gasche) [inline Changes]: Thanks, fixed. (I kept the attached "changelog" substantive.)

**Comment by gasche** [comment]
@whitequark, the CI on this PR seems to never terminate, while it is reported as successful on https://travis-ci.org/ocaml/ocamlbuild/builds/206057002 . Would you have any idea of what is happening? Maybe if Travis is unreliable, we should disable the "merge button disallowed while required CI runs are not complete" option?

**Comment by whitequark** [comment]
> Would you have any idea of what is happening?

Yes. Unfortunately, Amazon S3 is down in US-EAST-1, which caused internet-wide outage of any infrastructure that relies on it. This happens once every few years.

It's [recovering](status.aws.amazon.com) now, so let's just wait a bit.

**Comment by gasche** [comment]
Ok, the Changes will wait. Sorry for the noise.

**Pull request merged**